scripts/validate-front-matter/script.py

```diff
@@ -388,18 +388,22 @@ def validate(content, fpath, validation):
         ]
     }
     """
-    result = []
+
+    result = {
+        'num_front_matter_lines': 0,
+        'checks': [],
+    }
     num_lines = len(list(content.split("\n")))
 
     # Detect whether there is front matter
     if not content.endswith('\n'):
-        result.append({
+        result['checks'].append({
             'check': NO_TRAILING_NEWLINE,
             'line': num_lines, # last line
         })
         return result
```

Comment:
The actual bug was that here (among others), the returned result had a different structure than expected.